tests/unit/test_updater.py

```diff
@@ -1,40 +1,57 @@
 import json
 import os
+import mock
+import pytest
 
 from dvc import __version__
 from dvc.updater import Updater
 
 
-class MockResponse(object):
-    def __init__(self, json_data, status_code):
-        self.json_data = json_data
-        self.status_code = status_code
+@pytest.fixture
+def updater(dvc):
+    return Updater(dvc.dvc_dir)
 
-    def json(self):
-        return self.json_data
 
+@mock.patch("requests.get")
+def test_fetch(mock_get, updater):
+    mock_get.return_value.status_code = 200
+    mock_get.return_value.json.return_value = {"version": __version__}
 
-def mocked_requests_get(*args, **kwargs):
-    class MockResponse:
-        def __init__(self, json_data, status_code):
-            self.json_data = json_data
-            self.status_code = status_code
-
-        def json(self):
-            return self.json_data
-
-    return MockResponse({"version": __version__}, 200)
-
-
-def test_fetch(dvc_repo, mocker):
-    updater = Updater(dvc_repo.dvc_dir)
     assert not os.path.exists(updater.updater_file)
 
-    mock_get = mocker.patch("requests.get", side_effect=mocked_requests_get)
     updater.fetch(detach=False)
-    mock_get.assert_called_once_with(Updater.URL, timeout=Updater.TIMEOUT_GET)
 
+    mock_get.assert_called_once_with(Updater.URL, timeout=Updater.TIMEOUT_GET)
     assert os.path.isfile(updater.updater_file)
+
     with open(updater.updater_file, "r") as fobj:
         info = json.load(fobj)
+
     assert info["version"] == __version__
+
+
+@pytest.mark.parametrize(
+    "latest, current, result",
+    [
+        ("0.20.8", "0.21.0", False),
+        ("0.20.8", "0.20.8", False),
+        ("0.20.8", "0.19.0", True),
+    ],
+)
+def test_is_outdated(latest, current, result, updater):
+    updater.latest = latest
+    updater.current = current
+
+    assert updater._is_outdated() == result
+
+
+@pytest.mark.skipif(
+    os.getenv("TRAVIS_EVENT_TYPE") != "cron",
+    reason = "Only run on travis CRON to avoid generating too much logs"
+def test_check(updater, monkeypatch):
+    monkeypatch.setenv("DVC_TEST", "true")
```

Comment:
It's already `"true"`.

Reply:
> There is also a DVC_TEST env var

Then I didn't get this comment, @Suor

Reply:
You are setting it to `true`, but it is already `true`. Basically test is noop since I changed Updater to not run in tests several months ago. 

To fix this properly you need:
- add some mock to check that `updater.check()` is actually doing something
- fix it not doing something by adjusting env